packages/fork-choice/src/forkChoice/interface.ts

```diff
@@ -93,6 +94,11 @@ export interface IForkChoice {
   getHeadRoot(): RootHex;
   getHead(): ProtoBlock;
   updateHead(): ProtoBlock;
```

Comment:
now `beacon-node` only call `updateAndGetHead()`, I think we should drop this from the interface to avoid misusing it later